### PR TITLE
EDU-3009: Update Global Admin references to include new Account Owner role

### PR DESCRIPTION
**Pull request by fairlydurable**
- Audit site
- Find and apply changes

Make sure all changes are correct and appropriate

### CODE REVIEW

File: docs/production-deployment/cloud/account-setup/users.mdx [hunk 1]
```diff
@@ -251,7 +251,7 @@ The following table provides the API details associated with each account-level
 
 :::note
 
-Account Global Admin has Namespace Admin permissions on Namespaces.
+Account Owners and Global Admins have Namespace Admin permissions on Namespaces.
```
Comment (captainbeardo) [inline docs/production-deployment/cloud/account-setup/users.mdx:254]: suggestion: add "permissions on all Namespaces"

**Review by captainbeardo — APPROVED** [review]

File: docs/production-deployment/cloud/account-setup/users.mdx (hunk 1 above)
Comment (fairlydurable) [inline docs/production-deployment/cloud/account-setup/users.mdx:254]:
```suggestion
Account Owners and Global Admins have Namespace Admin permissions on all Namespaces.
```

Comment (fairlydurable) [reply docs/production-deployment/cloud/account-setup/users.mdx:254]: Adopting as suggested. Thank you.

File: docs/production-deployment/cloud/api-keys.mdx [hunk 2]
```diff
@@ -78,8 +78,8 @@ For instructions on managing API keys for your organization, see [Global Adminis
 API keys support both users and Service Accounts.
 Here are the differences in their permissions:
 
-- Both normal users and Global Administrators/Account Owners can create, delete, and update API key access using the Cloud UI or `tcld`.
-- Only Global Administrators/Account Owners can create, delete, and update access to API keys for Service Accounts.
+- Both normal users and Global Administrators and Account Owners can create, delete, and update API key access using the Cloud UI or `tcld`.
```
Comment (fairlydurable) [inline docs/production-deployment/cloud/api-keys.mdx:81]:
```suggestion
- Normal users, Global Administrators, and Account Owners can create, delete, and update API key access using the Cloud UI or `tcld`.
```

File: docs/production-deployment/cloud/get-started.mdx [hunk 3]
```diff
@@ -2,7 +2,7 @@
 id: get-started
 title: Get started with Temporal Cloud
 sidebar_label: Get started
-description: Start using Temporal Cloud by signing up, accepting Global Admin permissions, setting up CA certificates, creating a Namespace, inviting users, and connecting your Temporal Client and Workers.
+description: Start using Temporal Cloud by signing up, having the Account Owner or Global Admin role, setting up CA certificates, creating a Namespace, inviting users, and connecting your Temporal Client and Workers.
```
Comment (fairlydurable) [inline docs/production-deployment/cloud/get-started.mdx:5]:
```suggestion
description: Get started with Temporal Cloud. Sign up, verify your Account Owner or Global Admin role, set up CA certificates, create a Namespace, invite users, and connect your Temporal Client and Workers.
```

**Review by napcs — APPROVED** [review]

**Pull request merged**